kazel/config.go

```diff
@@ -33,11 +33,17 @@ type Cfg struct {
 	AddSourcesRules bool
 	// whether to have multiple build files in vendor/ or just one.
 	VendorMultipleBuildFiles bool
-	// whether to manage kubernetes' pkg/generated/openapi.
```

Comment:
I completely ripped out the existing openapi generator support, but I suppose I could separate these into two separate steps.